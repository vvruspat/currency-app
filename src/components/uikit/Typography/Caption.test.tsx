import { render, screen } from "@testing-library/react";
import { Caption } from ".";

test("renders caption", () => {
  render(<Caption>Test caption</Caption>, {});
  const captionElement = screen.getByText(/Test caption/i);
  expect(captionElement).toBeInTheDocument();
  expect(captionElement).toMatchSnapshot();
});
