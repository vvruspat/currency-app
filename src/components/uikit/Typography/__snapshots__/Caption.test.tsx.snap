// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders caption 1`] = `
<div
  class="caption"
>
  Test caption
</div>
`;
